pkg/bgpv1/manager/reconcile.go

```diff
@@ -378,7 +379,7 @@ func exportPodCIDRReconciler(
 		ctx:       ctx,
 		name:      "pod CIDR",
 		component: "manager.exportPodCIDRReconciler",
-		enabled:   newc.ExportPodCIDR,
+		enabled:   pointer.BoolDeref(newc.ExportPodCIDR, false),
```

Comment:
Should the default value of ExportPodCIDR be explicitly defined in API? ( No strong opinion on this )

Reply:
Yeah not sure either, but as this filed was released in a previous Cilium version, I would prefer the more conservative change to it (so, no?).

Reply:
From [Changing The API - On Compatibility](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#on-compatibility): "An API change is considered compatible if it: does not change existing semantics, including the semantic meaning of default values and behavior"

But then, our BGP API is still alpha, which means we in theory could do [almost any changes](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions) - but I guess we don't want it.

Reply:
Is it an incompatible change to set the default to `false` here? If not, I'd prefer to set it for consistency.

Reply:
Done in https://github.com/cilium/cilium/pull/26040/commits/4a064eb83a2fd1070bfdff21f41f0a1669e3ad17

Reply:
Done in https://github.com/cilium/cilium/pull/26040/commits/aa3458df112c4d33516e0e281679b3a264c8062a